charts/sonarr/values.yaml

```diff
@@ -25,6 +25,27 @@ service:
 ingress:
   enabled: false
 
+probes:
+  liveness:
+    enabled: true
+    ## Set this to true if you wish to specify your own livenessProbe
+    custom: true
+    ## The spec field contains the values for the default livenessProbe.
+    ## If you selected custom: true, this field holds the definition of the livenessProbe.
+    spec:
+      exec:
+        command:
+        - /usr/bin/env
+        - bash
+        - -c
+        - curl --fail localhost:8989/api/v3/system/status?apiKey=`IFS=\> && while
```

Comment:
Shouldn't this use target service and not the pod?

Reply:
What would be the benefit of targeting the service? Since the healthcheck is on the pod itself I don't see why it would be better.

Reply:
Shoot, i'm mistaken indeed.